util/src/main/java/google/registry/util/CloudTasksUtils.java

```diff
@@ -106,30 +106,34 @@ private static Task createTask(
     checkArgument(
         path != null && !path.isEmpty() && path.charAt(0) == '/',
         "The path must start with a '/'.");
+    checkArgument(
+        method.equals(HttpMethod.GET) || method.equals(HttpMethod.POST),
+        String.format("HTTP method %s is used. Only GET and POST are allowed.", method));
```

Comment:
You don't need to use `String.format` inside a `checkArgument` call.  There's an overload of `checkArgument()` that already takes a format string on its own.